main.tf

```diff
@@ -48,11 +48,19 @@ resource "aws_s3_bucket_cors_configuration" "main" {
   }
 }
 
+resource "aws_s3_bucket_ownership_controls" "main" {
```

Comment:
The ability to set this should be on the `README.md` features list ;)

Reply:
Done